src/clj/rems/db/applications.clj

```diff
@@ -327,7 +327,8 @@
 
 (defn get-approvals [user-id]
   (->> (get-applications-impl-batch user-id {})
-       (filterv (partial can-approve? user-id))))
+       (filterv (partial can-approve? user-id))
+       (map #(dissoc % :invitation-tokens)))) ; TODO use same hiding in all applications lists
```

Comment:
yes, please fix this TODO

Reply:
Doesn't make sense to fix now as this should all be part of v2 including a nice `get-applications-impl-batch` replacement.

Reply:
ok